pytest.ini

```diff
@@ -8,3 +8,4 @@ norecursedirs = .* docs completion refactor absolute_import namespace_package sc
 # fine as long as we are using `clean_jedi_cache` as a session scoped
 # fixture.
 usefixtures = clean_jedi_cache
+testpaths = test
```

Comment:
Fixed by moving `clean_jedi_cache` to top-level `conftest.py`.  Now there are two pieces of code that manage temporary directories for cache: as a session-level fixture and in `pytest_configure`/`pytest_unconfigure` hooks. I'm not sure which one is better to stay, so left them both in for now.